### ISSUE TITLE
Feature join-ad-interest-group is not enabled by Permissions Policy

### ISSUE THREAD

**Issue by alexm92** [GoogleChromeLabs/protected-audience-demo#15]
Started to see this error message on the advertiser.
`Uncaught (in promise) DOMException: Failed to execute 'joinAdInterestGroup' on 'Navigator': Feature join-ad-interest-group is not enabled by Permissions Policy`

A similar message is on the publisher, but for `run-ad-auction`

<img width="1680" alt="image" src="https://github.com/GoogleChromeLabs/protected-audience-demo/assets/2988929/7b5f5aa3-0683-40fc-8578-b0fccffbb8ef">

**Comment by dmdabbs** [comment]
@alexm92 on all Chrome versions (Canary, Dev, Beta, Stable) I see these warnings, not an exception: 

![image](https://github.com/GoogleChromeLabs/protected-audience-demo/assets/175532/2e525c45-50e7-46a2-b690-f06aae96f93f)

I tried to use Chrome's "header overrides" feature to add the Permissions-Policy header to the advertiser page response to grant the DSP permission and eliminate that warning, but it didn't work. I guess PAAPI doesn't see the overridden, additional header:

![image](https://github.com/GoogleChromeLabs/protected-audience-demo/assets/175532/0e8eb497-f35f-40eb-92fc-3dce941df0b3)

Google folks, consider adding the Permissions-Policy header for one of the embedded sites' iframes as an example, perhaps also with pointer to https://developer.chrome.com/en/docs/privacy-sandbox/permissions-policy/.

**Comment by dmdabbs** [comment]
DOH, one also needs to add the `allow` attribute to the iframe.

Adding it (via response override)

```
<iframe allow='join-ad-interest-group' id='adtech-pixel' src='https://protected-audience-demo-adtech.web.app'>
```
did not eliminate the error.

**Comment by alexm92** [comment]
@dmdabbs Maybe I'm in one of the experiments. For me, it's an error, not a warning.

I've cloned the demo repository and run it under a domain that I control. Everything worked well until a few days ago when the permission policy got enforced (for me).

I've tried both adding `Permissions-Policy: join-ad-interest-group=(self "https://protected-audience-demo-dsp.web.app")` and `<iframe allow="join-ad-interest-group" src="https://protected-audience-demo-adtech.web.app"></iframe>`. Neither seems to work.

Additionally, the SSP code does not run in an iframe, but directly on the publisher's page.
